wiki/Rules/Song_Content_Rules/id.md

```diff
@@ -1,32 +1,12 @@
----
-outdated: true
----
-
 # Aturan konten musik
 
 Dengan banyaknya variasi musik yang tersedia, sebagian besar lagu akan baik-baik saja untuk digunakan di osu!, asalkan tidak:
 
-- Sangat eksplisit, menjurus, atau sangat konfrontatif dan berhubungan dengan konten yang mengejutkan, cabul, atau sangat seksual
-- Mengangkat, mendukung, atau mendorong penyalahgunaan narkoba, bunuh diri, pembunuhan, atau melukai diri sendiri
-- Mendukung penekanan dan perpecahan terhadap rasial
-- Mengandung suara atau arti sangat keras dan tak tertahankan atau terlalu terpotong
 - Mengandung hal yang sangat politis
+- Mendukung tekanan dan perpecahan terhadap rasial
+- Mengandung volume lagu sangat keras dan tak tertahankan atau terlalu durasi lagu terpotong terlalu banyak
```

Comment:
yeah it's the same thing here -- in audio-related terms the word `clipped` here doesn't literally translate into `an audio file that's being cut` orz. instead it's referring to `an audio file that's excessively loud` (http://en.wikipedia.org/wiki/Clipping_(audio)) if any

--> `Memiliki volume lagu yang dengan sengaja dipasang sedemikian keras hingga memunculkan efek *clipping* yang berlebihan`

Reply:
Ah I thought clipping is the same as trimming